src/bin/cratesfyi.rs

```diff
@@ -298,6 +317,28 @@ pub fn main() {
                 priority,
             )
             .expect("Could not add crate to queue");
+        } else if let Some(matches) = matches.subcommand_matches("set") {
```

Comment:
This doesn't work - it checks for `set` in the `queue` namespace, not in the `default-priority` namespace. You'll need to do something like `if let Some(matches) = matches.subcommand_matches("default-priority") {` and then move this whole if let inside that block.